transport/transport.go

```diff
@@ -23,14 +24,15 @@ func NewHandler(us model.UserService) *Handler {
 	}
 
 	h.Mux.Use(httpx.RequestIDMiddleware(RequestID))
-	h.Mux.Use(httpx.LoggingMiddleware(utils.NewServerLogger("logging")))
-	h.Mux.Use(httpx.RecoverMiddleware(utils.NewServerLogger("recover")))
+	h.Mux.Use(httpx.LoggingMiddleware(slog.Default().WithGroup("logger")))
+	h.Mux.Use(httpx.RecoverMiddleware(slog.Default().WithGroup("recover")))
 	h.Mux.Use(h.auth)
 
 	h.Mux.Get("/admin/users", h.requireAdminUser(h.getUsers))
 	h.Mux.Route("/admin/users/{id}", func(r chi.Router) {
-		r.Get("/", h.requireAdminUser(h.handleUser))
-		r.Delete("/", h.requireAdminUser(h.handleUser))
+		r.Use(h.HandleResourceID("id", utils.ContextSetUserID))
```

Comment:
Instead of having handleUser handler, getUser and deleteUser gets user id from context.